### PR TITLE
feature:add we_chat

### PR DESCRIPTION
**Pull request by xiekeyi98**
增加向微信公众号群发消息的功能。

### CODE REVIEW

File: src/send.py [hunk 1]
```diff
@@ -101,6 +102,35 @@ def Send_email(txt, to_addr_str, subject):
...
+# 获取微信access_token
+# TODO 每两小时token会过期，一天大概只能获取2000次
+def get_token():
+    payload_access_token={
+        'grant_type': configs.GRANT_TYPE,
+        'appid': configs.APPID,
+        'secret': configs.SECRET
+    }
+    token_url='https://api.weixin.qq.com/cgi-bin/token'
+    r=requests.get(token_url,params=payload_access_token)
```
Comment (ZhaoQi99) [inline src/send.py]: It seems that you don't handle the exception which may occur.

Comment (xiekeyi98) [reply src/send.py]: Do you mean what I do just now ?

File: src/configs.py [hunk 2]
```diff
@@ -46,3 +46,10 @@
         'coding': ''
     }
 ]
+
+# we_chat config
+
+GRANT_TYPE = 'client_credentials'
```
Comment (xiekeyi98) [inline src/configs.py]: 添加默认参数类型。

File: src/configs.py [hunk 3]
```diff
@@ -49,7 +49,7 @@
 
 # we_chat config
 
-GRANT_TYPE = 'client_credentials'
+GRANT_TYPE = 'client_credential'
```
Comment (xiekeyi98) [inline src/configs.py:52]: I am very sorry for my carelessness. Shame on me .

Comment (ZhaoQi99) [reply src/configs.py:52]: It doesn't matter.

**Comment by xiekeyi98** [comment]
[图片]
image

Wonderful,It looks good

**Pull request merged**